examples/firecrawl_enhanced_config_example.ts

```diff
@@ -0,0 +1,232 @@
+/**
```

Comment:
same comment re: createReactAgent

Reply:
Changed

Reply:
seems like it's still using a full `StateGraph` here - i think it's fine to keep at most one example using full state graph (and we already do in  
`examples/filesystem_langgraph_example.ts`) but would keep the rest using `createReactAgent`